lib/CppInterOp/CppInterOp.cpp

```diff
@@ -855,8 +855,21 @@ static void GetClassDecls(TCppScope_t klass,
     if (auto* MD = dyn_cast<DeclType>(DI))
       methods.push_back(MD);
     else if (auto* USD = dyn_cast<UsingShadowDecl>(DI))
```

Comment:
I guess we could simplify the levels of indentation if we did:
```cpp
for (Decl* DI : CXXRD->decls()) {
  if (auto* USD = dyn_cast<UsingShadowDecl>(DI))
    DI = USD->getTargetDecl();
  ...
```

Reply:
No, we cannot do it, because we require the original `DI` later, in line 862.

Reply:
Oh, ok.